Week2/solution week2/week2solution-checkpoint.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1c539c92",
+   "metadata": {},
+   "source": [
+    "1.Create table stud(id,name) (use I'd as primary key) stud_details(id,city,mobileno) (Use I'd as foreign key)"
```

Comment:
why we need foreign key in second table ?

Reply:
to connect table data with another table